jsonnet/kube-prometheus/components/kube-state-metrics.libsonnet

```diff
@@ -11,7 +11,21 @@ local defaults = {
     requests: { cpu: '10m', memory: '190Mi' },
     limits: { cpu: '100m', memory: '250Mi' },
   },
-
+  kubeRbacProxyMain: {
+    resources+: {
+      limits+: { cpu: '40m' },
+      requests+: { cpu: '20m' },
+    },
+    limits+: { cpu: '40m' },
+    requests+: { cpu: '20m' },
+    name: 'kube-rbac-proxy-main',
+    upstream: 'http://127.0.0.1:8081/',
+    secureListenAddress: ':8443',
+    ports: [
+      { name: 'https-main', containerPort: 8443 },
+    ],
+    image: $.kubeRbacProxyImage,
```

Comment:
Due to how jsonnet can override things, we should set this in L102 and not here.